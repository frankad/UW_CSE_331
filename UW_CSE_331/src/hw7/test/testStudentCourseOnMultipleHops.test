# Test for a case to find the minimum cost path with multiple hops or differnt
# possible options to move from the start to destination node using the full
# studentInCourse dataset.

# build graph
LoadGraph studentInCourse studentInCourse.tsv

# test to find the minimum cost path, findMinimumCostPath
FindPath studentInCourse Olivia	Abigal