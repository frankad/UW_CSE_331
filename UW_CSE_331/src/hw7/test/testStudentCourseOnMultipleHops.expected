# Test for a case to find the minimum cost path with multiple hops or differnt
# possible options to move from the start to destination node using the full
# studentInCourse dataset.

# build graph
loaded graph studentInCourse

# test to find the minimum cost path, findMinimumCostPath
path from Olivia to Abigal:
Olivia to Chris with weight 1.000
Chris to Sam with weight 1.000
Sam to Liya with weight 1.000
Liya to Sophia with weight 1.000
Sophia to Abigal with weight 1.000
total cost: 5.000